ledger/src/blockstore_processor.rs

```diff
@@ -676,6 +677,7 @@ fn process_next_slots(
                 bank.slot(),
                 allocated.since(initial_allocation)
             );
+            *slot_count += 1;
```

Comment:
This seems like the wrong place to increment this as the slot has not yet been processed, and we already increment the slot count in `process_pending_slots` after `process_single_slot()` is called

Reply:
Yea, you're right. I realize it's just creating the bank here and not processing it.